src/modules/SearchLocation/MapOverlay/MoveMapGuide.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+
+import { useReastorageValue } from "@reastorage/react";
+
+import { locationStorage } from "src/store/location";
+import { Box } from "src/ui/Box";
+import { Typography } from "src/ui/Typography";
+
+export const MoveMapGuide = () => {
+  const { jibunAddress, roadAddress } = useReastorageValue(locationStorage);
+  const [guideText, setGuideTest] = useState(
+    "지도를 움직여 주소지를 설정해주세요.",
+  );
+  useEffect(() => {
+    if (jibunAddress || roadAddress) {
+      setGuideTest("입력하신 주소지를 확인해주세요");
+    }
+  }, [jibunAddress, roadAddress]);
```

Comment:
이 부분은 state로 처리되어야 할 이유가 없을 것 같아요~
```suggestion
const guideText = jibunAddress || roadAddress ? "입력하신 주소지를 확인해주세요" : "지도를 움직여 주소지를 설정해주세요."
```